airflow/plugins/operators/airtable_to_warehouse.py

```diff
@@ -0,0 +1,124 @@
+import pandas as pd
+import os
+
+from pyairtable import Table
+from calitp import save_to_gcfs, to_snakecase, write_table
+
+from airflow.models import BaseOperator
+
+
+def airtable_to_df(
+    air_base_id,
+    air_table_name,
+    id_name="__id",
+    rename_fields=None,
+    column_prefix=None,
+    api_key=None,
+):
```

Comment:
**thought:** This feels like a good candidate for a simple automated test. It doesn't look like we have any automated code test patterns set up in data-infra; something to add to the list.